### PR TITLE
[Fix](bangc-ops): update the code style of bangc-ops.

### PR DESCRIPTION
**Pull request by DanieeelLiu**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

update the code style of part files.
## 2. Modification

modified:   bangc-ops/kernels/ball_query/ball_query.h
	modified:   bangc-ops/kernels/ball_query/ball_query_union1.mlu
	modified:   bangc-ops/kernels/binary_op/binary_op_host.cpp
	modified:   bangc-ops/kernels/copy/copy.cpp
	modified:   bangc-ops/kernels/generate_proposals_v2/generate_proposals_v2.mlu
	modified:   bangc-ops/kernels/tensor_stride_process/tensor_stride_process.h
	modified:   bangc-ops/kernels/three_interpolate_forward/three_interpolate.cpp
	modified:   bangc-ops/kernels/three_interpolate_forward/three_interpolate_union1.mlu
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/include/api_test_tools.h
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/prior_box/prior_box_general.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/psroipool_backward/psroipool_backward.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/psroipool_backward/psroipool_backward_general.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/psroipool_forward/psroipool_forward.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/psroipool_forward/psroipool_forward_general.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/roi_crop_backward/roi_crop_backward.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/roi_crop_backward/roi_crop_backward_general.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/roi_crop_forward/roi_crop_forward.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/roi_crop_forward/roi_crop_forward_general.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/yolo_box/yolo_box.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/yolo_box/yolo_box_general.cpp
	modified:   bangc-ops/test/mlu_op_gtest/api_gtest/src/main.cpp
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest-death-test.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest-message.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest-param-test.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest-printers.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest-spi.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest-test-part.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest-typed-test.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest_pred_impl.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/gtest_prod.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-death-test-internal.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-filepath.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-internal.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-linked_ptr.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-param-util-generated.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-param-util.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-port.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-string.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-tuple.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/include/gtest/internal/gtest-type-util.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/src/gtest-death-test.cc
	modified:   bangc-ops/test/mlu_op_gtest/googletest/src/gtest-filepath.cc
	modified:   bangc-ops/test/mlu_op_gtest/googletest/src/gtest-internal-inl.h
	modified:   bangc-ops/test/mlu_op_gtest/googletest/src/gtest-port.cc
	modified:   bangc-ops/test/mlu_op_gtest/googletest/src/gtest-printers.cc
	modified:   bangc-ops/test/mlu_op_gtest/googletest/src/gtest-test-part.cc
	modified:   bangc-ops/test/mlu_op_gtest/googletest/src/gtest-typed-test.cc
	modified:   bangc-ops/test/mlu_op_gtest/googletest/src/gtest.cc
	modified:   bangc-ops/test/mlu_op_gtest/include/tools.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/include/evaluator.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/include/executor.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/include/memory_pool.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/include/parser.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/include/pb_test_tools.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/include/runtime.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/include/thread_pool.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/include/variable.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/case_collector.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/evaluator.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/executor.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/mlu_op_gtest.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/mlu_op_gtest.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/perf_test.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/perf_test.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/parser.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/pb_test_tools.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/runtime.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/ball_query/ball_query.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/ball_query/ball_query.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/generate_proposals_v2/generate_proposals_v2.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/generate_proposals_v2/generate_proposals_v2.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/generate_proposals_v2/generate_proposals_v2_impl.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/generate_proposals_v2/generate_proposals_v2_impl.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/poly_nms/pnms_impl.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/poly_nms/pnms_impl.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/poly_nms/poly_nms.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/prior_box/prior_box.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/prior_box/prior_box.h
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/psroipool_backward/psroipool_backward.cpp
	modified:   bangc-ops/test/mlu_op_gtest/pb_gtest/src/zoo/yolo_box/yolo_box.cpp
	modified:   bangc-ops/test/mlu_op_gtest/tools/pb2prototxt.cpp
	modified:   bangc-ops/test/mlu_op_gtest/tools/prototxt2pb.cpp
	modified:   bangc-ops/test/version_check/version_check.cpp
	modified:   tools/pre-commit

### CODE REVIEW

File: bangc-ops/kernels/three_interpolate_forward/three_interpolate_union1.mlu [hunk 1]
```diff
@@ -79,11 +79,11 @@ __mlu_global__ void MLUKernelThreeInterpolateForward(
   /*
    * NRAM partition
    *  |-----------------------------------------------------------------------------------|
-   *  |           nram_features                  |        nram_features_transpose         |
+   *  |           nram_features                  | nram_features_transpose |
```
Comment (ArtIntAI) [inline bangc-ops/kernels/three_interpolate_forward/three_interpolate_union1.mlu:82]: 这里空间划分的注释需要恢复到之前的格式（同样的长度代表同样的大小）

**Review by WangYuan-cn — APPROVED** [review]

File: bangc-ops/kernels/three_interpolate_forward/three_interpolate_union1.mlu (hunk 1 above)
Comment (DanieeelLiu) [reply bangc-ops/kernels/three_interpolate_forward/three_interpolate_union1.mlu]: 已恢复

[the remark this answers is not in the document]
Comment (DanieeelLiu) [reply bangc-ops/kernels/three_interpolate_forward/three_interpolate_union1.mlu]: 已恢复

File: bangc-ops/test/mlu_op_gtest/pb_gtest/include/evaluator.h [hunk 2]
```diff
@@ -55,10 +55,12 @@ class Evaluator {
...
 
   struct Criterion {
-    Criterion(Formula f, double t, bool e = true) : formula(f), threshold(t), enable(e) {}
+    Criterion(Formula f, double t, bool e = true)
+        : formula(f), threshold(t), enable(e) {}
     Formula formula;
     double threshold = 0;
-    bool enable      = true;  // if false, only compute it, but won't mark case failed.
+    bool enable =
+        true;  // if false, only compute it, but won't mark case failed.
```
Comment (wushaoqiang2168) [inline bangc-ops/test/mlu_op_gtest/pb_gtest/include/evaluator.h:63]: 此处可以把注释放到上面，这样就不会换行了

File: bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/mlu_op_gtest.cpp [hunk 3]
```diff
@@ -220,16 +224,18 @@ void TestSuite::ThreadX() {
   auto set_device = [](std::shared_ptr<Context> ctx) {
     std::lock_guard<std::mutex> lk(ctx->mtx);
     auto it = ctx->been_initialized.find(std::this_thread::get_id());
-    if (it == ctx->been_initialized.end()) {  // if current thread has not been set device.
+    if (it == ctx->been_initialized
+                  .end()) {  // if current thread has not been set device.
```
Comment (wushaoqiang2168) [inline bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/mlu_op_gtest.cpp:228]: 此处注释可以放到if上面

File: bangc-ops/test/mlu_op_gtest/pb_gtest/include/evaluator.h [hunk 4]
```diff
@@ -103,16 +102,16 @@ class Evaluator {
...
-  double computeDiff3_2(float *baseline_result, float *mlu_result, size_t count);
+  double computeDiff3_2(float *baseline_result, float *mlu_result,
+                        size_t count);
   double computeDiff4(float *baseline_result, float *mlu_result, size_t count);
   inline std::string showFormula(Formula f);
 
-  std::vector<Criterion> criterion_vec_;  // vector of (diff1+thresdhold) /(diff2 + threshold)
-  std::vector<ErrorWrap> error_vec_;      // vetor output's error
+  std::vector<Criterion>
+      criterion_vec_;  // vector of (diff1+thresdhold) /(diff2 + threshold)
```
Comment (wushaoqiang2168) [inline bangc-ops/test/mlu_op_gtest/pb_gtest/include/evaluator.h:113]: 注释放上面

**Review by LittleReal — APPROVED** [review]
api_gtest is ok

File: bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/mlu_op_gtest.cpp [hunk 5]
```diff
@@ -31,7 +31,8 @@ std::string TestSuite::op_name_ = "";  // NOLINT
 std::vector<std::string> TestSuite::case_path_vec_ = {};
 std::shared_ptr<mluoptest::ExecuteConfig> TestSuite::ecfg_ =
     std::make_shared<mluoptest::ExecuteConfig>();
-std::shared_ptr<mluoptest::ExecuteContext> TestSuite::ectx_ = nullptr;  // depends on thread num.
+std::shared_ptr<mluoptest::ExecuteContext> TestSuite::ectx_ =
+    nullptr;  // depends on thread num.
```
[the remark this answers is not in the document]
Comment (DanieeelLiu) [reply bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/mlu_op_gtest.cpp]: 已修改

File: bangc-ops/test/mlu_op_gtest/pb_gtest/include/variable.h [hunk 6]
```diff
@@ -29,27 +29,29 @@
...
   std::string get_vmpeak_ = "";
   TestSummary summary_;
 
-  int dev_id_     = 0;      // the picked device id, make sure gtest run on the picked device.
-  int rand_n_     = -1;     // pick n * random case, -1 for uninitialized
-  int repeat_     = 1;      // perf-repeat repeat * kernel enqueue cnrtQueue_t, and get ave hw_time
-  int thread_num_ = 1;      // thread num
-  bool shuffle_   = false;  // shuffle cases.
+  int dev_id_ =
+      0;  // the picked device id, make sure gtest run on the picked device.
```
[the remark this answers is not in the document]
Comment (DanieeelLiu) [reply bangc-ops/test/mlu_op_gtest/pb_gtest/include/variable.h]: 已修改

File: bangc-ops/test/mlu_op_gtest/pb_gtest/include/evaluator.h (hunk 4 above)
Comment (DanieeelLiu) [reply bangc-ops/test/mlu_op_gtest/pb_gtest/include/evaluator.h]: 已修改

File: bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/mlu_op_gtest.cpp (hunk 3 above)
Comment (DanieeelLiu) [reply bangc-ops/test/mlu_op_gtest/pb_gtest/src/gtest/mlu_op_gtest.cpp]: 已修改

File: bangc-ops/test/mlu_op_gtest/pb_gtest/include/evaluator.h (hunk 2 above)
Comment (DanieeelLiu) [reply bangc-ops/test/mlu_op_gtest/pb_gtest/include/evaluator.h]: 已修改

**Review by wushaoqiang2168 — APPROVED** [review]

**Pull request merged**